apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -154,13 +179,15 @@ class SectionProgress extends Component {
               <VirtualizedDetailView
                 section={section}
                 scriptData={scriptData}
+                onScroll={this.afterScroll}
               />
               <ProgressLegend
                 excludeCsfColumn={scriptData.excludeCsfColumnInLegend}
               />
             </div>
           }
         </div>
+        {this.renderTooltips()}
```

Comment:
I'm not sure this will be better, but we already have `levelDataInitialized` in the render function, so you could have 

```
{`levelDataInitialized` && this.renderTooltips()}
```
instead of just checking on script data like you did on line 93.